lib/src/generateInit.cc

```diff
@@ -358,6 +358,26 @@ unsigned int genInitializeDeviceKernel(CodeStream &os, const NNmodel &model, int
                                 }
                             }
                         }
+                        // Loop through current sources
+                        if (n.second.isInjected()) {
```

Comment:
I don't really think the ``NeuronGroup::isInjected`` method is really necessary at all - if there's nothing in the vector then nothing happens

Reply:
right